src/homework3.py

```diff
@@ -0,0 +1,84 @@
+import sys
+
+from pyspark import StorageLevel
+from pyspark.sql import SparkSession
+
+from transformer import Rolling100DayTransform
+
+spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+database = "baseball"
+user = "root"
+password = ""
+server = "localhost"
+port = 3306
+
+jdbc_url = f"jdbc:mysql://{server}:{port}/{database}?permitMysqlScheme"
```

Comment:
good flexible coding here, easy for different user to use by only changing the attributes above instead of modifying this line.